liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/BaseDevTest.java

```diff
@@ -144,16 +146,23 @@ private static void stopProcess(boolean isDevMode) throws IOException, Interrupt
       // shut down dev mode
       if (writer != null) {
          if(isDevMode) {
-            writer.write("exit"); // trigger dev mode to shut down
+            writer.write("exit\n"); // trigger dev mode to shut down
          }
          else {
             process.destroy(); // stop run
          }
          writer.flush();
          writer.close();
 
+         try {
+            process.waitFor(120, TimeUnit.SECONDS);
+            process.exitValue();
+         } catch (IllegalThreadStateException e) {
+            throw e;
```

Comment:
What is the purpose of catching this exception if you just rethrow it directly?

Reply:
I'll fix that